.github/workflows/tests.yml

```diff
@@ -81,3 +90,49 @@ jobs:
           cardId: ${{ github.event.client_payload.addComment.cardId }}
           comment: ${{ github.event.client_payload.addComment.comment }}
 
+  checkAndAddTag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: check if tag exists
+        id: hasTag
+        uses: ./hasTag
+        with:
+          host: ${{ github.event.client_payload.common.host }}
+          apiToken: ${{ github.event.client_payload.common.apiToken }}
+          cardId: ${{ github.event.client_payload.addComment.cardId }}
```

Comment:
I guess it doesn't really matter, but shouldn't this be `client_payload.hasTag`, rather than `...addComment`? Or maybe boardId and cardId should be considered `common` because they are used so much.

Reply:
I moved the `cardId` to `common`.